### DISCUSSION TITLE
Why isn't the Test tab presented as trees like the Build tab?

### DISCUSSION THREAD

**Discussion by yhgu2000** [microsoft/vscode-cmake-tools#3567] [Q&A] [answered]
I have many tests in my project. They are all listed together:

![图片](https://github.com/microsoft/vscode-cmake-tools/assets/49391101/d0f94a42-6904-4be2-9ff7-73d82281a284)

Why aren't them shown as trees, like in the Build tab?

![图片](https://github.com/microsoft/vscode-cmake-tools/assets/49391101/4bf827e3-abe7-4d41-8075-fddbaac27045)

**Comment by gcampbell-msft** [comment, marked as answer]
@yhgu2000 There is a feature that now allows people to have more of a tree view. https://github.com/microsoft/vscode-cmake-tools/pull/3759
